src/administrative-sdk/connection/connection-controller.js

```diff
@@ -140,7 +141,12 @@ export default class Connection {
       self._session = null;
       self.fireEvent('websocketClosed');
```

Comment:
I noticed previous notations of 'webSocket'. Here 'websocket' is used. Maybe use webSocket for consistency?